modules/installation-special-config-storage.adoc

```diff
@@ -164,3 +163,8 @@ $ butane $HOME/clusterconfig/worker-storage.bu -o ./99-worker-storage.yaml
 . Save the Butane config in case you need to update the manifest in the future.
 
 . Continue with the remainder of the {product-title} deployment.
+
+[IMPORTANT]
+====
+If you configure additional data partitions, they are only encrypted when encryption is explicitly requested.
```

Comment:
Suggestion to strengthen the language a bit:

```suggestion
If you configure additional data partitions, they will not be encrypted unless encryption is explicitly requested.
```

Reply:
Thanks, updated to your original wording 😀

Reply:
...whoops, I didn't even notice. :sweat_smile: